src/AnchoredOverlay/AnchoredOverlay.tsx

```diff
@@ -104,11 +104,16 @@ export const AnchoredOverlay: React.FC<AnchoredOverlayProps> = ({
   )
   const onAnchorClick = useCallback(
     (event: React.MouseEvent<HTMLElement>) => {
-      if (!event.defaultPrevented && event.button === 0 && !open) {
+      if (event.defaultPrevented || event.button !== 0) {
```

Comment:
@dmarcey had mentioned an issue with double clicking the anchor causing the menu to immediately open and close.  I think we could avoid this by adding a simple `|| event.detail === 2` here or `|| event.detail > 1`.  Playing around with it locally, I could see users getting frustrated by triple clicking not causing any change, so `=== 2` might be the safer bet.  Thoughts?

Reply:
I’m not adamant about this, but adding special handling for double-clicks to the library feels like a code smell to me, since double-clicking is uncommon on the web. This PR makes it possible to customize double-click behavior in consumer code—I added [a story demonstrating this](https://github.com/primer/components/blob/5feb1ce19d47689be6f772719b280f2ea4e2d166/src/stories/ActionMenu.stories.tsx#L277-L322).

Reply:
I agree that this is the right solution.  Thanks for adding a story to show exactly how to implement that on the consumer side!